src/main.rs

```diff
@@ -78,28 +115,33 @@ struct HTTPResult {
 async fn handle(
     request: http::Request<Vec<u8>>,
     mut response: simple_server::ResponseBuilder,
+    force_canister_id: &str,
```

Comment:
Passing an `&Option<str>` or something like that might be nicer here (see above)

Reply:
Same response as above. Rust is new to me, and I couldn't figure out how to work properly with optionals, but I COULD get this version to compile. I would love to have done it your way, just couldn't figure out how.